support/insights-operator/disabling-insights-operator.adoc

```diff
@@ -0,0 +1,23 @@
+[id="disabling-insights-operator"]
+= Disabling the Insights Operator
+include::modules/common-attributes.adoc[]
+:context: insights-operator-disabling
+
+toc::[]
+
+:FeatureName: The Insights Operator
+include::modules/technology-preview.adoc[leveloffset=+0]
+
+By default, the Insights Operator is enabled for {product-title}. To disable data collection by the Insights Operator, you must:
+
+. xref:../../support/insights-operator/disabling-insights-operator.html#insights-operator-new-pull-secret_insights-operator-disabling[Modify the global cluster pull secret] to disable Telemetry and the Insights Operator.
```

Comment:
I don't think you need these extra xrefs. If you keep them, use .adoc instead of .html.